lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
Amazing, thanks for thoroughly testing this out!!

Instead of generating the initializer, I'm going to set `ASSET_SYNC_INCLUDE_WEBPACKER_ASSETS=true` in production, since that aligns with the rest of our `asset_sync` setup (see `heroku config -a fashionthing-production | grep ASSET_SYNC`). Sorry, I totally forgot to mention that was needed to get this working!

I'm ok with running `rake assets:sync` explicitly. Where do I put that? In the `circle.yml`?